app/src/screens/recovery/SaveRecoveryPhraseScreen.tsx

```
@@ -12,19 +11,15 @@ import { Title } from '../../components/typography/Title';
 
-type NextScreen = keyof RootStackParamList;
-interface SaveRecoveryPhraseScreenProps
-  extends StaticScreenProps<{ nextScreen?: NextScreen } | undefined> {}
+interface SaveRecoveryPhraseScreenProps {}
 
-const SaveRecoveryPhraseScreen: React.FC<SaveRecoveryPhraseScreenProps> = ({
-  route,
-}) => {
+const SaveRecoveryPhraseScreen: React.FC<
+  SaveRecoveryPhraseScreenProps
+> = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)